truffle-config.js

```diff
@@ -31,7 +31,7 @@ module.exports = {
    * will spin up a development blockchain for you on port 9545 when you
    * run `develop` or `test`. You can ask a truffle command to use a specific
    * network from the command line, e.g
-   *
+   *  npx ganache-cli -p 8545 && truffle migrate -network development
```

Comment:
you can add this as a script to package.json file. This will enable other devs to easily start ganache by running something like `npm run start:ganache`

Reply:
okay noted and done